rosdep/python.yaml

```
@@ -1216,6 +1216,11 @@ python-control-pip:
   ubuntu:
     pip:
       packages: [control]
+python3-cookiecutter:
+  debian:
+    buster: [python3-cookiecutter]
+    stretch: [python3-cookiecutter]
+  ubuntu: [python3-cookiecutter]
```

Two suggestions here:
1. This package [isn't available for Ubuntu Focal](https://packages.ubuntu.com/search?keywords=python3-cookiecutter&searchon=names&suite=all&section=all). That platform will need to be specifically excluded.
2. This package is available for Fedora, as is suggested by the automation: https://packages.fedoraproject.org/pkgs/python-cookiecutter/python3-cookiecutter/

```suggestion
  fedora: [python3-cookiecutter]
  ubuntu:
    '*': [python3-cookiecutter]
    focal: null
```